integration/tests/main/03-create-private-package-repository.spec.js

```diff
@@ -105,12 +105,17 @@ test("Create a new private package repository successfully", async ({ page }) =>
   // so that it get loaded in the DOM when using the toContainText assert
   await page.locator(".values-editor div.modified").click({ button: "right" });
   await page.locator("text=Command Palette").click();
-  await page.locator('[aria-label="Type to narrow down results\\."]').click();
-  await page.locator('[aria-label="Type to narrow down results\\."]').fill(">find");
-  await page.locator('label:has-text("FindCtrl+F")').click();
-  await page.locator('[aria-label="Find"]').fill("tag: 2.4.48");
-  // Note the U+200C, which is a zero-width non-joiner, character instead of a space
-  await expect(page.locator(".values-editor div.modified")).toContainText("tag:·‌2.4.48-debian-10-r75");
```

Comment:
We finally were able to remove the U+200C character and use a normal space instead thanks to this Monaco upgrade!